spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -288,6 +291,26 @@ private static Integer obtainManagementServerPort(ServerBuilder serverBuilder,
         }
     }
 
+    @Nullable
+    private static Integer getExposedInternalServicePort(BeanFactory beanFactory) {
+        final Object internalServices = findBean(beanFactory, INTERNAL_SERVICES_CLASS);
+        if (internalServices == null) {
+            return null;
+        }
+        try {
+            final boolean actuatorEnabled = (boolean) ACTUATOR_ENABLED_METHOD.invoke(internalServices);
```

Comment:
nit+optional; I'm wondering if we can determine if the user enabled actuator by declaring a dependency for `ArmeriaSettings` (at `actuatorServerConfigurator`)

By doing so, we can avoid declaring a specific public API and remove a reflective call.
Not a strong opinion though 😄

Reply:
Thanks for your comment.😄
I'm not sure about how to implement this `actuatorServerConfigurator` method  looks like. 🤔

The current implementation is like below.
- spring-boot-actuator is enabled automatically if it has this dependency.
- actuator end-point is exposed in only internal port if `actuator` id is specified in the property of internal-services.
Like 
```
armeria:
  internal-services:
    include: metrics, docs, actuator <-- actuator is exposed in only 18080 port.
    port: 18080
```

Reply:
I guess he meant to use `armeriaSettings.getInternalServices().getInclude()` to determine `actuatorEnabled`.

Reply:
A possible implementation I had in mind was..

```
ArmeriaServerConfigurator actuatorServerConfigurator(
        WebEndpointsSupplier endpointsSupplier,
        ...
        ArmeriaSettings armeriaSettings) {
    ...
    final Integer internalServicePort = getExposedInternalServicePort(beanFactory, armeriaSettings);
    ...
}
```

```
private static Integer getExposedInternalServicePort(BeanFactory beanFactory, ArmeriaSettings armeriaSettings) {
    ...
    boolean actuatorEnabled = false;
    final InternalServiceProperties internalServiceProperties = armeriaSettings.getInternalServices();
    if (internalServiceProperties != null && internalServiceProperties.getInclude() != null) {
        actuatorEnabled = internalServiceProperties.getInclude().contains(InternalServiceId.ACTUATOR) ||
                          internalServiceProperties.getInclude().contains(InternalServiceId.ALL);
    }
    if (!actuatorEnabled) {
        return null;
    }
    ...
}
```

Reply:
Thanks for answering. 😄 I see. It looks good. Let me fix the codes more.